pkg/apis/core/validation/seed_test.go

```diff
@@ -324,21 +324,21 @@ var _ = Describe("Seed Validation Tests", func() {
 		})
 
 		It("should forbid Seed with overlap to default vpn range (subset)", func() {
-			shootDefaultPodCIDR := "192.168.123.128/28"     // 192.168.123.128 -> 192.168.123.143
+			shootDefaultPodCIDR := "192.168.124.0/23"       // 192.168.124.0-> 192.168.125.0
```

Comment:
`192.168.124.0/23` starts with `192.168.124.0`, but ends with `192.168.125.255`. Therefore, please adapt the comment accordingly.